### PR TITLE
Correct validation loss logging message

### PR DESCRIPTION
**Pull request by trung-vt**
There is a small error in the logging command where the training loss value is reported as the validation loss instead.

### CODE REVIEW

**Comment by epapoutsellis** [comment]
thank you @trung-vt.
